src/gbcc_mbc.c

```
@@ -97,9 +97,10 @@ uint8_t gbcc_mbc_mbc2_read(struct gbc *gbc, uint16_t addr) {
 void gbcc_mbc_mbc2_write(struct gbc *gbc, uint16_t addr, uint8_t val) {
 }
 uint8_t gbcc_mbc_mbc3_read(struct gbc *gbc, uint16_t addr) {
-	return 0;
```

Ignoring errors does not fix them